pimd/pim_cmd.c

```diff
@@ -5717,16 +5755,11 @@ static void pim_cmd_show_ip_multicast_helper(struct pim_instance *pim,
 
 	show_scan_oil_stats(pim, vty, now);
 
-	show_multicast_interfaces(pim, vty);
+	show_multicast_interfaces(pim, vty, false);
 }
 
-DEFUN (show_ip_multicast,
-       show_ip_multicast_cmd,
-       "show ip multicast [vrf NAME]",
-       SHOW_STR
-       IP_STR
-       VRF_CMD_HELP_STR
-       "Multicast global information\n")
+DEFUN(show_ip_multicast, show_ip_multicast_cmd, "show ip multicast [vrf NAME]",
```

Comment:
I doubt this change.

Reply:
Click for style suggestions from polychaeta

-DEFUN (show_ip_multicast,
-       show_ip_multicast_cmd,
-       "show ip multicast [vrf NAME]",
-       SHOW_STR
-       IP_STR
-       "Multicast global information\n"
-       VRF_CMD_HELP_STR)
+DEFUN(show_ip_multicast, show_ip_multicast_cmd, "show ip multicast [vrf NAME]",
+      SHOW_STR IP_STR "Multicast global information\n" VRF_CMD_HELP_STR)